Makefile

```diff
@@ -39,12 +39,12 @@ distclean:
 .PHONY: check
 
 test: build
-ifeq ("$(BAP_RUN_TEST)","true")
+ifneq ("$(BAP_NO_TESTS)","true")
```

Comment:
are we using this environment variables at all? If not, then let's just drop this ifelses from the Makefile.

Reply:
yes, it's not hard to get rid of them

Reply:
So let's drop them, and also let's try to add `-al` just curious if it will work.